server/routes/routerRequestHandlers/checkExemptionExistsInPrisonerContext.ts

```diff
@@ -4,13 +4,13 @@ import { getPrisonerContext } from '../../data/session/prisonerContexts'
 /**
  * Request handler function to check the exemption data exists in the prisoner context.
  */
-const checkExemptionExistsInPrisonerContext = async (req: Request, res: Response, next: NextFunction) => {
-  if (!getPrisonerContext(req.session, req.params.prisonNumber).reviewExemptionForm) {
+const checkExemptionDtoExistsInPrisonerContext = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
For consistency and clarity I think this function should be called `checkReviewExemptionDtoExistsInPrisonerContext` and the filename should be `checkReviewExemptionDtoExistsInPrisonerContext.ts`

(if someone finds this source file, from looking at the filename and/or function name, they wont have the context about how/where it is used. They will see "check exemption dto exists....", but wont have the context about what the "exemption" is all about)